flink-table-store-format/pom.xml

```diff
@@ -190,6 +190,13 @@ under the License.
                 </exclusion>
             </exclusions>
         </dependency>
+
+        <dependency>
+            <groupId>junit</groupId>
```

Comment:
Do we need this dependency? I see you are using junit 5.

Reply:
deleted